### ISSUE TITLE
UI waits for sound to render making tuning a slow process

### ISSUE THREAD

**Issue by HitCoder9768** [stakira/OpenUtau#467]
### Acknowledgement

- [x] I have read Getting-Started and FAQ

### 🐛 Describe the bug

While working on projects there is a consistent issue of OpenUtau hanging every time a tuning parameter is changed, so that the UI can wait for the sample to render. This makes it impossible to tune efficiently, especially on hardware that isn't extremely new or powerful.

### Explains how to reproduce the bug

- Tune or modify a USTx in any context on any mid-to-low-end hardware

### OS & Version

Linux 5.15 (Arch Linux, EndeavourOS)

### Logs

[log20220915.txt](https://github.com/stakira/OpenUtau/files/9577086/log20220915.txt)

**Comment by stakira** [comment]
you can turn off pre-render in preferences

**Comment by oxygen-dioxide** [comment]
> Even with pre-render switched off, I end up with performance issues though they only appear upon hitting play.
> 
> It can be a tad irritating when I hit play and the playhead does not follow the sound output. I know for many people this can be negligible but I find it hard to follow what the sound equates to on the piano roll.
> 
> These two aspects should, in the long run, ideally become asynchronous from one another. Obviously it's not a high priority but i feel that it may be a good QoL improvement for future updates.

Which renderer or engine are you using? I recommend wordline-r for note and pitch editing, which doesn't call external process and is efficient.
